src/include/daos_obj.h

```diff
@@ -347,6 +347,52 @@ daos_obj_generate_id(daos_obj_id_t *oid, daos_ofeat_t ofeats,
 	oid->hi |= hdr;
 }
 
+/** Flags for oclass hints */
+enum {
+	/** Flags to control OC Redundancy */
+	DAOS_OC_RDD_DEF		= (1 << 0),	/** Default - use RF prop */
+	DAOS_OC_RDD_NO		= (1 << 1),	/** No redundancy */
```

Comment:
What happens if someone specify RDD_NO and has a RF > 0 ?

Reply:
my assumption is that you get no redundancy

Reply:
should we allow user to specify higher redundancy level which is higher than the default, or user should just choose specific object class in this case?

Reply:
sorry i read the question wrong first.
i think the expectation here is that if we specify insufficient redundancy level, the generate API would fail.
if more, we should allow it i think.